sk0-back/src/users/dtos/create-user.dto.ts

```diff
@@ -1,5 +1,7 @@
 import {
   IsEmail,
+  IsNumber,
+  IsOptional,
```

Comment:
I am not sure if you are using those in the file.

Reply:
Yes, I just corrected it, because the tests kept failing :)